site/src/pages/OrganizationSettingsPage/OrganizationProvisionerKeysPage/OrganizationProvisionerKeysPageView.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+	type ProvisionerKeyDaemons,
+	ProvisionerKeyIDBuiltIn,
+	ProvisionerKeyIDPSK,
+	ProvisionerKeyIDUserAuth,
+} from "api/typesGenerated";
+import { Button } from "components/Button/Button";
+import { EmptyState } from "components/EmptyState/EmptyState";
+import { Link } from "components/Link/Link";
+import { Loader } from "components/Loader/Loader";
+import { Paywall } from "components/Paywall/Paywall";
+import {
+	SettingsHeader,
+	SettingsHeaderDescription,
+	SettingsHeaderTitle,
+} from "components/SettingsHeader/SettingsHeader";
+import {
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableRow,
+} from "components/Table/Table";
+import type { FC } from "react";
+import { docs } from "utils/docs";
+import { ProvisionerKeyRow } from "./ProvisionerKeyRow";
+
+interface OrganizationProvisionerKeysPageViewProps {
+	showPaywall: boolean | undefined;
+	provisionerKeyDaemons: ProvisionerKeyDaemons[] | undefined;
+	buildVersion: string | undefined;
+	error: unknown;
+	onRetry: () => void;
+}
+
+export const OrganizationProvisionerKeysPageView: FC<
+	OrganizationProvisionerKeysPageViewProps
+> = ({ showPaywall, provisionerKeyDaemons, buildVersion, error, onRetry }) => {
+	return (
+		<section>
+			<SettingsHeader>
+				<SettingsHeaderTitle>Provisioner Keys</SettingsHeaderTitle>
+				<SettingsHeaderDescription>
+					Manage provisioner keys used to authenticate provisioner instances.{" "}
+					<Link href={docs("/admin/provisioners")}>View docs</Link>
+				</SettingsHeaderDescription>
+			</SettingsHeader>
+
+			{showPaywall ? (
+				<Paywall
+					message="Provisioners"
+					description="Provisioners run your Terraform to create templates and workspaces. You need a Premium license to use this feature for multiple organizations."
```

Comment:
Not just multi-org right? Keys themselves require license too.

Reply:
This is a copy-paste of the existing paywall: https://github.com/coder/coder/blob/main/site/src/pages/OrganizationSettingsPage/OrganizationProvisionersPage/OrganizationProvisionersPageView.tsx#L100-L102